CONTRIBUTING.md

```diff
@@ -20,8 +20,8 @@
 [license-url]: https://github.com/tldr-pages/tldr/blob/master/LICENSE.md
 [license-image]: https://img.shields.io/github/license/tldr-pages/tldr.svg
 
-Contributions to the tldr-pages project are [most welcome](GOVERNANCE.md)!
-All `tldr` pages are stored in Markdown right here on GitHub.
+Contributions to the TLDR-pages project are [most welcome](GOVERNANCE.md)!
```

Comment:
I'm unsure about this. 
Should we have this in uppercase? @sbrl @mebeim @waldyrious

Reply:
Not really sure.. but this suggested change is inconsistent: it only changes some occurrences of the word, not all of them. :thinking:. I don't personally have a preference between the two by the way.

Reply:
I think I have an opinion now.
"TLDR" is the term while "tldr" is the project name. When we're talking about the project, I believe we should keep it lower case.

Reply:
That could be a good point.

Reply:
I believe that a while ago it was decided that `tldr-pages` is the official wording for referring to the tldr-pages project.

Reply:
> I believe that a while ago it was decided that tldr-pages is the official wording for referring to the tldr-pages project.

IIRC we didn't end up making a final decision. We just recently restarted discussion in #1109, but let me know if there are other discussions I'm missing.

In any case, I'm pretty sure all-caps TLDR was never widely supported in such discussions, so this particular bit should be reverted.